test/unit/org/connectionTest.ts

```diff
@@ -235,8 +239,11 @@ describe('Connection', () => {
 
     try {
       await shouldThrow(conn.autoFetchQuery('TEST_SOQL'));
-    } catch (err) {
-      expect(err.message).to.equal(errorMsg);
+    } catch (e) {
+      if (!(e instanceof Error)) {
```

Comment:
there's a whole lot of this pattern in the tests now.  It lets you be more specific about what the error is (Error, SfError, etc) before asserting props on it.